tools/devtool

```diff
@@ -603,13 +612,15 @@ apply_performance_tweaks() {
 
 unapply_performance_tweaks() {
   # reenable turbo boost
-  sudo sh -c "echo 0 > /sys/devices/system/cpu/intel_pstate/no_turbo" &> /dev/null
-
-  # We do not reset the governor, as keeping track of each CPUs configured governor is not trivial here. On our CI
-  # instances, the performance governor is current the default anyway (2023/11/14)
+  if [[ -d /sys/devices/system/cpu/intel_pstate ]]; then
+    sudo sh -c "echo 0 > /sys/devices/system/cpu/intel_pstate/no_turbo" &> /dev/null
+  fi
```

Comment:
Here, we should unapply the `min_perf_pct` and `max_perf_pct` configurations as well.

Reply:
unless something else runs afterwards. Right now, I can't think of anything that would be affected by this, other than these tests, but I feel a bit nervous about us adding something in the future and that gets affected in a way we don't understand because we will have forgotten this

Reply:
Per offline discussion: The only thing stopping me from doing this was my lack of understanding of bash variable scopes. Fixed now :)

Reply:
thanks for that Patrick